server.js

```diff
@@ -93,23 +94,46 @@ async function setAllowedOrigin() {
   }
 }
 
-log.info('Starting CDAP UI ...');
+log.info('Starting CDAP UI ...', 'goose');
```

Comment:
goose?

Reply:
I normally log `goose` places because its extremely unlikely that word is used elsewhere